cli.py

```diff
@@ -6,7 +6,9 @@
 
 # Options for command line interface
 @click.command()
-@click.argument('dfs_site', nargs=-1)
+@click.argument('dfs_site',
+                required=True,
```

Comment:
makes it so you have to pass in at least one argument

Reply:
Fixes issues of bypassing dfs site argument before getting to prompt.